sdk/containerregistry/container-registry/README.md

```diff
@@ -31,21 +48,67 @@ npm install @azure/container-registry
 
-### Further examples
+### Authenticate the client
+
+The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.
+
+```javascript
+const { ContainerRegistryClient } = require("@azure/container-registry");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const endpoint = process.env.REGISTRY_ENDPOINT;
+// Create a ContainerRegistryClient that will authenticate through Active Directory
+const client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+```
+
+Note that these samples assume you have a `REGISTRY_ENDPOINT` environment variable set, which is the URL including the name of the login server and the `https://` prefix.
 
-Top-level examples usually include things like creating and authenticating the main Client. If your service supports multiple means of authenticating (e.g. key-based and Azure Active Directory) you can give a separate example of each.
+For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).
 
 ## Key concepts
 
-### ContainerRegistryClient
+A **registry** stores Docker images and [OCI Artifacts](https://opencontainers.org/). An image or artifact consists of a **manifest** and **layers**. An image's manifest describes the layers that make up the image, and is uniquely identified by its **digest**. An image can also be "tagged" to give it a human-readable alias. An image or artifact can have zero or more **tags** associated with it, and each tag uniquely identifies the image. A collection of images that share the same name but have different tags, is referred to as a **repository**.
 
-`ContainerRegistryClient` provides operations to interact with an Azure Container Registry instance.
+For more information please see [Container Registry Concepts](https://docs.microsoft.com/azure/container-registry/container-registry-concepts).
 
 ## Examples
 
-### First Example
+### Listing repositories
 
-<!-- Examples should showcase the primary, or "champion" scenarios of the client SDK. -->
+Iterate through the collection of repositories in the registry.
+
+```javascript
+const { ContainerRegistryClient } = require("@azure/container-registry");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+async function main() {
+  // Get the service endpoint from the environment
+  const endpoint = process.env.REGISTRY_ENDPOINT;
+  // Create a new ContainerRegistryClient
+  const client = new ContainerRegistryClient(endpoint, new DefaultAzureCredential());
+
+  console.log("Listing repositories");
+  const iterator = client.listRepositories();
+  for await (const repository of iterator) {
+    console.log(`  repository: ${repository}`);
+  }
+
+  console.log("  by pages");
+  const pages = client.listRepositories().byPage({ maxPageSize: 2 });
```

Comment:
My understanding is that paging through a pageable by pages instead of single items is an advanced scenario.  This is true at least for .NET.  If it's true for TS as well, could the sample not iterate by pages?

Reply:
We have both listing items, and listing by pages in almost all of our listing samples so I will keep it here.

Reply:
okay this is readme. I will remove byPage()